symposion/schedule/admin.py

```diff
@@ -103,5 +103,15 @@ def tutorial_max(self, presentation):
         return getattr(presentation.proposal, 'max_attendees', 'N/A')
     tutorial_max.short_description = 'Attendees Max'
 
+    # When a presentation is saved in the admin we send an email to the staff
+    def save_model(self, request, obj, form, change):
+        send_email(
+            ["pycon-staff@python.org"],
+            "presentation_updated",
+            context={"presentation": obj}
+        )
+        # And also save the object as the admin requested
+        obj.save()
```

Comment:
Maybe either do the save first, or wrap the send_email in a try/except, so that if something goes wrong with the send_email, the changes aren't lost?